geoportailv3/static/js/mymaps/mymapsdirective.js

```diff
@@ -38,6 +39,114 @@ app.module.directive('appMymaps', app.mymapsDirective);
  * @constructor
  * @export
  */
-app.MymapsDirectiveController = function() {};
+app.MymapsDirectiveController = function() {
+
+  /**
+   * @type {string}
+   * @private
+   */
+  this.defaultTitle_ = 'Map without title';
+
+  /**
+   * @type {boolean}
+   * @export
+   */
+  this.modifying = false;
+
+  /**
+   * @type {boolean}
+   * @export
+   */
+  this.choosing = false;
+
+  /**
+   * @type {Object?}
+   * @export
+   *
+   * TODO Declare a more detailed type.
+   */
+  this.mymap = null;
+
+  /**
+   * String to be used in the title field in the modifying window.
+   * Helps canceling modifications.
+   * @type {string}
```

Comment:
You're missing `@export` here. This is important as `newTitle` is used in the template.